cpp/src/hash/murmur_hash.cu

```diff
@@ -51,11 +52,21 @@ std::unique_ptr<column> murmur_hash3_32(table_view const& input,
   thrust::tabulate(rmm::exec_policy(stream),
                    output_view.begin<hash_value_type>(),
                    output_view.end<hash_value_type>(),
-                   row_hasher.device_hasher<cudf::detail::MurmurHash3_32>(nullable, seed));
+                   row_hasher.device_hasher<MurmurHash3_32>(nullable, seed));
 
   return output;
 }
 
 }  // namespace detail
+
+std::unique_ptr<column> murmur_hash3_32(table_view const& input,
```

Comment:
As above, let’s pick a consistent name and choice of where underscores go. I understand we may have case differences between a class and function but the name and placement of underscores should be matched.